packages/material-ui/scripts/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+
+const input = './src/index.js';
+
+const name = 'material-ui';
+
+const globals = {
+  react: 'React',
+  'react-dom': 'ReactDOM',
+};
+
+const getBabelOptions = () => ({
+  exclude: /node_modules/,
+  runtimeHelpers: true,
```

Comment:
What is this option for? I can't find it on React side https://github.com/facebook/react/blob/master/scripts/rollup/build.js.

> Alternatively, if you know what you're doing, you can use the transform-runtime plugin. If you do this, use runtimeHelpers: true:

https://github.com/rollup/rollup-plugin-babel#helpers

Reply:
Why you rely on react here? They has quite specific config. For libs it can be smaller. runtimeHelpers should allow to use babel-runtime instead of inlined helpers (which is default behaviour of this plugin).

Reply:
Should we be using babel-runtime instead?

Reply:
I'm surprised we have to provide an option to Rollup. Babel was taking care of it on its own with wepback.

Reply:
Default behavior of rollup plugin is smarter than webpack loader which inlines helpers in every file. Rollup plugin inlines them once.